packages/api/cms-api/src/mailer/mailer.service.ts

```diff
@@ -74,6 +75,9 @@ export class MailerService {
             await this.entityManager.flush();
```

Comment:
not ideal that the MailerService does flush as side effect. But the usual alternative (not flush in a service) is also not ideal, as it can be forgotten easily.

A solution might be one of these https://mikro-orm.io/docs/transactions#approach-2-explicitly which makes sure (If I understand correctly) the mail log is always saved, and nothing else is flushed as side effect.